File: leet_1365_count_smaller_numbers.py
class Solution:
    def smallerNumbersThanCurrent(self, nums: List[int]) -> List[int]:
        res = [0] * len(nums)
        for i in range(len(nums)):
            for n in nums:
                if n < nums[i]:
                    res[i] += 1
        return res

File: leet_1365_count_smaller_numbers.cpp
class Solution {
public:
    vector<int> smallerNumbersThanCurrent(vector<int>& nums)
    {
        vector<int> res(nums.size());
        int i = -1, j;

        while (++i < nums.size())
        {
            for (int n : nums)
                if (n < nums[i])
                    res[i]++;
        }
        return res;
    }
};
